styles.css

```diff
@@ -305,4 +318,112 @@ main {
   position: absolute;
   margin-left: 180px;
   margin-top: 187px;
-}
\ No newline at end of file
+}
+
+.formsection {
+  display: grid;
+  grid-template-columns: 320px;
+  grid-template-rows: 830px;
+  margin-top: 16px;
+}
+
+.getstarted {
+  position: absolute;
+  width: 249px;
+  height: 50px;
+  left: 16px;
+  margin-top: 80px;
+  font-family: 'Playfair Display', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 45px;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+.main {
+  position: absolute;
+  width: 288px;
+  height: 63px;
+  left: 16px;
+  margin-top: 130px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  color: #1a2236;
+}
+
+footer {
+  position: absolute;
+  width: 320px;
+  height: 150px;
+  left: 0;
+  margin-top: 4630px;
+  background: #1a2236;
+}
+
+.pa {
+  position: absolute;
+  width: 127px;
+  height: 24px;
+  left: 16px;
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #92dcdc;
+  margin-top: 30px;
+}
+
+.arrowbox {
+  position: absolute;
+  width: 22px;
+  height: 22px;
+  left: 140px;
+  margin-top: 32px;
+}
+
+.socialmediaicons {
+  margin-top: 90px;
+  display: flex;
+  justify-content: space-around;
+}
+
+.form {
+  margin-top: 230px;
+  margin-left: 16px;
+}
+
+.form form {
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 20px;
+}
+
+form input {
+  width: 288px;
+  height: 52px;
+}
+
+form textarea {
+  width: 288px;
+  height: 112px;
+}
+
+form #submit {
+  color: #fff;
+  background: #1a2236;
+  cursor: pointer;
+}
+
+form #submit:hover {
+  border: 5px solid #00f;
+}
```

Comment:
As requested by the previous reviewer, your project should be responsive and similar in all widths of the mobile version. However, it is currently not responsive since it has a fixed width which does not allow the sections to increase in width when you increase the width of the screen as shown below :point_down: thereby creating a white space to the right side. Kindly check and fix that. 


![Screenshot from 2023-01-25 08-12-10](https://user-images.githubusercontent.com/65233860/214512457-73384dd9-b9e7-437e-a5e3-f79355d9f753.png)


![Screenshot from 2023-01-25 08-12-26](https://user-images.githubusercontent.com/65233860/214512471-6663d3de-b274-4b72-b360-0ede81dcf625.png)

![Screenshot from 2023-01-25 08-12-35](https://user-images.githubusercontent.com/65233860/214512479-e578bb70-de53-4472-87b1-40110f0b8503.png)